ui/packages/catalog-ui-search/src/main/webapp/react-component/location/direction.js

```diff
@@ -14,10 +14,10 @@
  **/
 const React = require('react')
 
-const CustomElements = require('../../../js/CustomElements.js')
+const CustomElements = require('../../js/CustomElements.js')
 const Component = CustomElements.registerReact('direction')
 
-class Direction extends React.Component {
+class DirectionInput extends React.Component {
```

Comment:
There's an export also called `Direction` in `dms-utils`, and some files import both. When they import this `Direction` they call it `DirectionInput` in order to distinguish the two, so I figured it might reduce confusion to just rename this one to what it's called when it's imported

Reply:
makes sense 👍